numpy/core/src/multiarray/_multiarray_tests.c.src

```diff
@@ -656,6 +656,35 @@ npy_updateifcopy_deprecation(PyObject* NPY_UNUSED(self), PyObject* args)
     Py_RETURN_NONE;
 }
 
+/* used to test PyArray_As1D usage emits not implemented error */
+static PyObject*
+npy_pyarrayas1d_deprecation(PyObject* NPY_UNUSED(self), PyObject* NPY_UNUSED(args))
+{
+    PyObject **result = Py_BuildValue("i", 42);
+    int dim = 4;
+    double arg[2] = {1, 2};
+    int temp = PyArray_As1D(result, (char **)&arg, &dim, NPY_DOUBLE);
```

Comment:
Define `result` as `PyObject *` and use `&result` here. (and below) The shippable tests are just broken, but the Travis failures are real due to compiler warnings.

Reply:
My bad, I overwrote Matti's changes by force-pushing the commit without pulling :(